src/sdk/ConversationTranslator/Conversation.ts

```diff
@@ -95,19 +128,28 @@ export class ConversationImpl extends Conversation implements IDisposable {
     private privProperties: PropertyCollection;
```

Comment:
The ConversationImpl class should be moved out of the src/sdk folder since it is not intended to be directly invoked by end users.

Reply:
Code updated per suggestion.